Spanner/src/TransactionConfigurationTrait.php

```diff
@@ -63,6 +63,34 @@ private function transactionSelector(array &$options, array $previous = [])
         ];
     }
 
+    /**
+     * Configure the DirectedReadOptions.
```

Comment:
```suggestion
     * Configure the DirectedReadOptions.
     * May not return the user-provided DirectedReadOptions if it is malformed
     * or unsupported. Such cases may occur in case of a read-write transaction.
```

Reply:
@ajupazhamayil this suggested doc change might not be accurate, pls re-check